crates/bevy_input/src/mouse.rs

```diff
@@ -155,3 +156,79 @@ pub fn mouse_button_input_system(
         }
     }
 }
+
+/// Tracks how much the mouse has moved every frame
+/// This resource is reset to zero every frame
+///
+/// This resource sums the total [`MouseMotion`] events received this frame.
+#[derive(Resource, Debug, Clone, Copy, PartialEq, Reflect, Default)]
+#[reflect(Debug, PartialEq)]
```

Comment:
We may also want to register `ReflectResource` for these types as well. They probably aren't super useful stored in scene files haha but this still allows them to be modified dynamically at runtime via reflection